tests/test.py

```diff
@@ -3,7 +3,7 @@
 import uuid
 
 import redis
-import common
+from tests import common
```

Comment:
So `python -m pytest tests/test.py` works with this change; what about `python tests/test.py`?  I actually have been using the latter to invoke this test and I think we should stick with `unittest` ;).  (The former was left there for historical reason.)

Reply:
Yep, that works too :)